### DISCUSSION TITLE
First time trying out simphony

### DISCUSSION THREAD

**Discussion by Argonghost** [BYUCamachoLab/simphony#86] [Q&A] [answered]
Hello,

  I'm actually a new user of Simphony - SiePIC via python.... and Im amazed by the library.  I've been following the pydocs simphony tutorial trying to recreate the example of the layout-aware MZI simulation exactly as per the attached python on the website and this is the error I keep getting;

  File "C:\Users\abdalla.hableel\Desktop\untitled3.py", line 84, in <module>
    c = mzi()

  File "C:\Users\abdalla.hableel\Desktop\untitled3.py", line 43, in mzi
    ysplit = c << y_splitter.component

AttributeError: 'YBranch' object has no attribute 'component'

Sorry but i couldn't find anyone else to reach out to.   I do have SiePIC and gdsfactory installed so im really stuck and unsure as to why i keep getting this error.  i copied the exact code as in the tutorial.  your help is appreciated.

**Comment by sequoiap** [comment]
@SkandanC

**Reply by Argonghost** [reply]
Hi
Has there been any update on the issue?

**Reply by sequoiap** [reply, marked as answer]
Hi @Argonghost,

In our latest refactor of simphony, we have decided to join forces with the excellent developers of the [SAX project](https://github.com/flaport/sax). We're switching simphony to use sax as a backend, while simphony continues to provide various model libraries (such as the SiEPIC model library as well as our machine-learning-trained SiPANN library) that can be used within the sax framework. Additionally, various quantum API's are soon to be released in simphony.

SAX offers the following:
* integration with gdsfactory
* monte-carlo simulations
* location-aware variation simulations
* along with much more.

We decided that, instead of independently developing two very similar projects, we might as well work together on one and expand its capabilities further. There are many plans yet for SAX. No further development is planned for the older versions of simphony. This would be my recommendation.

Sequoia
